### DISCUSSION TITLE
Theoretical back-testing

### DISCUSSION THREAD

**Discussion by foranpower** [OpenBB-finance/OpenBB#337] [Ideas]
Just dropping an idea here, no pressure to use it. Welcome all criticism and comments.

A back-tester. Stonk has portfolio optimization. So, this is piggy-backing off that for a Theoretical Tab.

Integrating an existing backtester would be cool. To see how one's strategy would have performed historically. Such as buying vertical option spreads based on bets and tesla tweets.

Not sure which backtester/live quant trading open source platforms are comfortable of partnering.
Suggesting QuantConnect because it's open source. Glad to ask them if it could help.

Of course other ideas welcome.

**Comment by jmaslek** [comment]
Love it. 

Been thinking about how to integrate it.  Quantconnect is open source so that could be one way.

I've also thought about other packages like backtrader that could do easier  strategies like sma crosses or rsi stuff.
